presto-tests/src/main/java/com/facebook/presto/tests/QueryAssertions.java

```diff
@@ -395,12 +395,9 @@ public static void copyTables(
             boolean ifNotExists,
             boolean bucketed)
     {
-        log.info("Loading data from %s.%s...", sourceCatalog, sourceSchema);
```

Comment:
I think having a loading log can be beneficial and i don't think there is that much pollution

Reply:
I am direct evidence of the opposite. I was trying to find failing tests and instead I was just scrolling through page after page of these messages. 

YAGNI!